src/kagglehub/auth.py

```diff
@@ -1,2 +1,14 @@
+import json
+from kagglehub.config import CREDENTIALS_JSON_KEY, CREDENTIALS_JSON_USERNAME, KaggleApiCredentials, _get_kaggle_credentials_file
+
+
 def login():
-    raise NotImplementedError()
+    """Prompt the user for their Kaggle username and API key and save them to a global variable."""
+
+    username = input("Enter your Kaggle username: ")
+    api_key = input("Enter your Kaggle API key: ")
+
+    global _kaggle_credentials
```

Comment:
I would keep the global and the variables in the `config` module.

Add a `set_kaggle_credentials` method to `config.py`. And define the global variables there.